recommenders/models/rbm/rbm.py

```diff
@@ -738,17 +638,17 @@ def fit(self, xtr, xtst):
             if self.with_metrics and i % self.display == 0:
                 log.info("training epoch %i rmse %f" % (i, epoch_tr_err))
 
-            Rmse_train.append(epoch_tr_err)  # mse training error per training epoch
+            rmse_train.append(epoch_tr_err)  # mse training error per training epoch
 
         # optionally evaluate precision metrics
         precision_train, precision_test = self.train_test_precision(xtst)
-        elapsed = self.time()
 
-        log.info("done training, Training time %f2" % elapsed)
+        # stop the timer
+        self.timer.stop()
 
-        self.display_metrics(Rmse_train, precision_train, precision_test)
+        log.info("done training, Training time %f2" % self.timer.interval)
 
-        return elapsed
+        self.display_metrics(rmse_train, precision_train, precision_test)
```

Comment:
same here, if we follow [single responsibility](https://github.com/Microsoft/Recommenders/wiki/Coding-Guidelines#single-responsibility) display_metrics should be outside the fit function. See previous comment for the reasoning and advantages of doing this

Reply:
Removed `display_metrics` function altogether. Instead, the RMSE is now plotted by calling the `line_graph` function in the `recommenders.utils.plot` module explicitly in the RBM notebooks.